### ISSUE TITLE
The cost calculation fails with an expression

### ISSUE THREAD

**Issue by AzizNgoueya** [Qiskit/qiskit-ibm-runtime#851]
**Describe the bug**
When assigning a value containing an operation in the Options().max_execution_time, I got an error `{"code":1208,"message":"Invalid job request body: Key: \\\'ProgramRunParams.Cost\\\' Error:Field validation for \\\'Cost\\\' failed on the \\\'min\\\' tag.","solution":"Check the JSON schema.","more_info":"https://cloud.ibm.com/apidocs/quantum-computing#error-handling"}`

**Steps to reproduce**

```
options = Options()
options.max_execution_time = 4*60
sampler = Sampler(session = backend, options = options)
circuit = QuantumCircuit(1,1)
circuit.h(0)
circuit.measure_all()
job = sampler.run(circuit, shots = 1024) 
```

**Expected behavior**
It should work

**Suggested solutions**

`options.max_execution_time = 240` works

**Additional Information**

- **qiskit-ibm-runtime version**: 0.9.3
- **Python version**: 3.8.16
- **Operating system**: Mac OS

**Comment by merav-aharoni** [comment]
`options.max_execution_time` must be at least 300 seconds. So actually both versions should fail. I suggest we improve the error message.

**Comment by kt474** [comment]
fixed by #853
